### PR TITLE
Mention the web build script in examples/README.md

### PR DESCRIPTION
**Pull request by ghost**
## Description

I think the `build.bash` script should be mentioned in the README.

## Related Issues

The build script tried to move resources from the nonexistent directory `examples/cube/resources/` so I corrected that too.

### CODE REVIEW

File: build.bash [hunk 1]
```diff
@@ -15,7 +15,7 @@ case $1 in
         RUSTFLAGS=--cfg=web_sys_unstable_apis cargo build --target wasm32-unknown-unknown $BUILD_FLAGS --bin $@
         mkdir -p target/generated/
         rm -rf target/generated/*
-        cp -r examples/$1/resources target/generated/ || true
+        cp -r examples/resources target/generated/ || true
```
Comment (cwfitzgerald) [inline build.bash]: This is intentional - some examples (notably scene-viewer) do have resources that need to be copied. The `|| true` lets this command fail, only copying the resources if they exist.

**Review by cwfitzgerald — COMMENTED** [review]
Note, otherwise good

**Review by cwfitzgerald — CHANGES_REQUESTED** [review]
Wrong button

File: build.bash (hunk 1 above)
Comment (ghost) [reply build.bash]: I undid that change, but now there're two commits and i have no idea about squashing and merging them unfortunately.

Comment (cwfitzgerald) [reply build.bash]: Github can squash for you, no worries.

Comment (ghost) [reply build.bash]: Oh, cool! I had a look but couldn't find anything.

Comment (ghost) [reply build.bash]: ~~Just tried again manually and fucked it up again; feel free to deny/cancel this pull request, i've no idea what i'm doing.~~

Oh no wait, it should be `git push --force` at the end! Every time i've tried it before the number of commits just got bigger...

**Pull request closed without merging**